packages/wrangler/src/__tests__/index.test.ts

```diff
@@ -80,15 +76,12 @@ describe("wrangler", () => {
           wrangler kv:bulk           💪 Interact with multiple Workers KV key-value pairs at once
           wrangler pages             ⚡️ Configure Cloudflare Pages
           wrangler r2                📦 Interact with an R2 store
-
```

Comment:
These new lines are actually included in the output. Did you mean to remove them?

Reply:
Hmm... OK OK, I think `\s\n` was also catching `\n\n`.

Reply:
Consider using a replacer function https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#specifying_a_function_as_a_parameter if you can't figure out a clean regex

Reply:
Thanks. The regex wasn't too hard: `/[^\S\n]+\n/`.